### DISCUSSION TITLE
Format Number

### DISCUSSION THREAD

**Discussion by duyhd** [apache/superset#29113] [Q&A / Help]
![image](https://github.com/apache/superset/assets/49530713/39d4eab2-5f3a-49c0-8fc9-8d371658b634)
I want convert format number from 226,049,182,616.69 to 226.049.182.616,69 
Can anyone help me

**Comment by sfirke** [comment]
See the discussion in the pull request that implemented this feature, specifically this post: https://github.com/apache/superset/pull/20075#issuecomment-1588176706.  You override the [D3_FORMAT part of `config.py`](https://github.com/apache/superset/blob/55d9d5711245db516629da3a5ea962e7ba0d321b/superset/config.py#L395) from your `superset_config.py` or other config settings file.

I expect if you read all of the posts in that PR you will be set but please reply if anything is still unclear. @duyhd, when you get it working, feel free to share your value of `D3_FORMAT` back here to help others who might be wondering the same thing.
